python_ta/typecheck/base.py

```diff
@@ -674,7 +688,7 @@ def can_unify(self, t1: type, t2: type) -> bool:
         return not isinstance(tc.unify(t1, t2, None), TypeFail)
 
     @accept_failable
-    def unify_call(self, func_var, *arg_types, node=None) -> TypeResult:
+    def unify_call(self, func_var: type, *arg_types: List[type], node: Optional[NodeNG] = None) -> TypeResult:
```

Comment:
Ah, so for varargs (like `arg_types`), you're supposed to only annotate the type of a single element: `type` instead of `List[type]`. You can keep this in mind when you tackle more complex function signatures (_later_).

ref: https://www.python.org/dev/peps/pep-0484/#arbitrary-argument-lists-and-default-argument-values